packages/migrate/migrations/package/index.js

```diff
@@ -0,0 +1,71 @@
+import fs from 'fs';
+import colors from 'kleur';
+import path from 'path';
+import prompts from 'prompts';
+import { pathToFileURL } from 'url';
+import { bail, check_git } from '../../utils.js';
+import { migrate_config } from './migrate_config.js';
+import { migrate_pkg } from './migrate_pkg.js';
+
+export async function migrate() {
+	if (!fs.existsSync('svelte.config.js')) {
+		bail('Please re-run this script in a directory with a svelte.config.js');
+	}
+	if (!fs.existsSync('package.json')) {
+		bail('Please re-run this script in a directory with a package.json');
+	}
+
+	const use_git = check_git();
+
+	const response = await prompts({
+		type: 'confirm',
+		name: 'value',
+		message: 'Continue?',
+		initial: false
+	});
```

Comment:
what are we confirming here? if 'Continue?' is the first thing someone sees when running `npx svelte-migrate package` it's a bit confusing

Reply:
Added a warning log prior to it (and there would also be a Git warning if your working directory isn't clean)